packages/components/select/demo/CustomLabel.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <ix-select v-model:value="singleValue" :options="options" multiple :multipleLimit="5" :maxLabelCount="3">
+    <template #customLabel="{ option }"> <ix-icon :name="option.value" /> {{ option.label }} </template>
+    <template #customMaxLabel="{ option }"> and {{ option.value.length }} more selected </template>
+  </ix-select>
+</template>
+<script lang="ts">
+import { defineComponent, ref } from 'vue'
+
+export default defineComponent({
+  setup() {
+    const singleValue = ref('github')
+
+    const options = [
+      { label: 'Github', value: 'github' },
+      { label: 'Gitlab', value: 'gitlab' },
+      { label: 'Chrome', value: 'chrome' },
+      { label: 'Edeg', value: 'edeg' },
+      { label: 'Ie', value: 'ie' },
```

Comment:
```suggestion
      { label: 'IE', value: 'ie' },
```
从英文的角度上来讲，英文的首字母缩写应该是采用大写的形式，不等同于英文第一个字母大写

Reply:
顺带补充一点，Ie，为马耳他语中的一个二合字母。 QAQ